curp/src/server/cmd_execute_worker.rs

```diff
@@ -1,198 +0,0 @@
-use std::sync::Arc;
```

Comment:
If you want to modify the name of a file, you can use `git mv` commanad. Compared to using `mv` to change the filename directly, `git mv` can preserves the file's diff infomation, which goes a long way towards reducing the burden of code review.